src/util/textInput.js

```diff
@@ -33,5 +33,8 @@ const onKeyCode = curry((modifier, keyCode, callback, event) =>
 // use like: <input type='text' onKeyDown={onEnter(callback)}/>
 export const onEnter = onKeyCode(null, keyMap.ENTER)
 export const onCmdEnter = onKeyCode('metaKey', keyMap.ENTER)
+export const onCtrlEnter = onKeyCode('ctrlKey', keyMap.ENTER)
+export const onCmdOrCtrlEnter = (callback, event) =>
```

Comment:
Let me know if you can see a point free way to write this that you'd like to use. I feel like there should be one, but rusty with this stuff.